src/vs/workbench/contrib/mergeEditor/browser/view/editors/resultCodeEditorView.ts

```diff
@@ -160,5 +168,19 @@ export class ResultCodeEditorView extends CodeEditorView {
 				);
 
 		}));
+
+
+		// title menu
+		const titleMenu = menuService.createMenu(MenuId.MergeInputResultToolbar, contextKeyService);
+		const toolBar = new ToolBar(this.htmlElements.toolbar, contextMenuService);
+		const toolBarUpdate = () => {
+			const secondary: IAction[] = [];
+			createAndFillInActionBarActions(titleMenu, { renderShortTitle: true }, secondary);
+			toolBar.setActions([], secondary);
+		};
+		this._store.add(toolBar);
+		this._store.add(titleMenu);
+		this._store.add(titleMenu.onDidChange(toolBarUpdate));
+		toolBarUpdate();
```

Comment:
This duplicates the menu in the input editors should now move into the super tye

Reply:
I think this is coincidence that all editors use the toolbar, not a semantic dependency. Thus, imo composition makes more sense to de-duplicate the code.